my_package/activelearning.py

```diff
@@ -63,22 +56,26 @@ def main(config):
         sampler = hydra.utils.instantiate(
             config.sampler,
             surrogate=surrogate,
+            device=device,
+            float_precision=float_precision,
+            _recursive_=False,
```

Comment:
What is this `_recursive_` parameter for and why is it needed here?

Reply:
recursive prevents hydra from instantiating subclasses too
e.g. if we specify gflownet, we also have env and plolicy classes etc.

Reply:
Is it necessary because within the sampler there are _hydra_ instantiations of other classes?